sdk-setup/src/rpmvalidation

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# Copyright (C) 2018 Jolla Ltd.
+# Contact: Ville Nummela <ville.nummela@jolla.com>
+#
+# This file is part of the SailfishOS SDK
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+usage() {
+    cat <<EOF
+Run a basic quality criteria check for the given RPM file.
+
+Usage: 
+    $0 -t <target> [OPTIONS] <rpm-file>
+	
+Options:
+    -t|--target <target> The OS version to validate for. 
+                         Accepts the same argument as 'sb2 -t' does.
+EOF
+}
+
+SUITE_INI_DIR=/usr/share/rpmvalidation
+SUITE_DIR=/usr/libexec/rpmvalidation/suites
+
+validators=()
+
+populate_validators() {
+    VALIDATOR_INIS=$(sb2 -t "$OPT_TARGET" ls "$SUITE_INI_DIR")
+    for f in $VALIDATOR_INIS; do
+        validators+=(${f%.ini})
+    done
+}
+
+validator_value() {
+    sb2 -t "$OPT_TARGET" python3 -c 'import sys, configparser; config = configparser.RawConfigParser(); config.read(sys.argv[2:]); sec, val = sys.argv[1].split("."); print(config[sec][val]);' RpmValidationSuite.$2 /usr/share/rpmvalidation/$1.ini
+}
+
+run_validator() {
+    validator_exec=$(validator_value $1 exec)
+    validator_cmd=$(echo "$validator_exec" | sed -e "s#%f#$RPM_FILE#" -)
+    sb2 -t "$OPT_TARGET" "$validator_cmd"
+}
+
+run_all_validators() {
+    echo "Using validation suites: ${validators[@]}"
+	for f in ${validators[@]}; do
```

Comment:
The expansion should be quoted `"${var[@]}"`, otherwise the "@" has no effect compared to "*". Also `f` should be declared as local beforehand. And it is a good convention to always initialize variable when declaring it as local - in some more complex cases the behavior is surprising so I learned to always initialize it.

```
local f=
for f in ...; do
```